playbooks/preflight-check.yml

```diff
@@ -87,6 +87,31 @@
   tags:
     - leap-fs-check
 
+- name: Gather lxc filesystem used percentages
+  shell: |
+    df -h {{ lxc_container_directory | default('/var/lib/lxc') }} | tail -n +2 | sed s/%//g | awk '{if($5 > 50) print "Alert: "$5 "% full - "$0}'
+  register: lxc_fs_check
+  when:
+    - inventory_host in groups["os-infra_hosts"]
+  tags:
+    - lxc-fs-check
+
+# RLM-1371 detect if enough space for rootfs backup
+- name: Check if lxc filesystem is greater than 50% full
+  debug:
+    msg: >-
+      The LXC filesystem(s) have been found to be greater than 50% full.
+      Details: {{ item.split() | join(' ') }}
+      If you have fully awareness of the potential risk having failure at rootfs backup stage,
+      you can skip this check with tag 'lxc-fs-check' by rerun 'preflight-check.yml'.
```

Comment:
Instead of skip-tag make a Boolean option a user can set with `-e` and then add it to the conditions of these two tasks.

``` yaml
- name: Gather lxc filesystem used percentages
  shell: |
    df -h {{ lxc_container_directory | default('/var/lib/lxc') }} | tail -n +2 | sed s/%//g | awk '{if($5 > 50) print "Alert: "$5 "% full - "$0}'
  register: lxc_fs_check
  when:
    - lxc_fs_check | default(false) | bool
    - inventory_host in groups["os-infra_hosts"]

# RLM-1371 detect if enough space for rootfs backup
- name: Check if lxc filesystem is greater than 50% full
  debug:
    msg: >-
      The LXC filesystem(s) have been found to be greater than 50% full.
      Details: {{ item.split() | join(' ') }}
      If you're fully aware of the potential risk, you can skip this check with
      by passing the option 'lxc_fs_check=yes'.
  failed_when: true
  when:
    - lxc_fs_check | default(false) | bool
    - inventory_host in groups["os-infra_hosts"]
    - lxc_fs_check.stdout_lines
  with_items: "{{ lxc_fs_check.stdout_lines }}"
```

Reply:
yeah, that's really a good point. But 'lxc_fs_check' both plays as return value and boolean flag as you gave. It looks a little bit complex and ambiguous, so can we give one separated flag variable to indicate skipping?